meteor/client/ui/Shelf/ShelfDashboardLayout.tsx

```diff
@@ -51,209 +59,195 @@ export interface IShelfDashboardLayoutProps {
 
 export function ShelfDashboardLayout(props: IShelfDashboardLayoutProps) {
 	const { rundownLayout } = props
+	const interpolableSource: InterpolatedPropsSource = {
```

Comment:
Looking at your implementation I don't think that this variable should be named `interpolableSource`. It just contains a Studio and a ShowStyleBase and that's what the name should reflect. Right now you have named your variable for what you intend to do with it, instead of what it actually is.

You wouldn't write:
```
someFunction(props) => {
  const interpolableSource: InterpoaltedPropsSource = {
    studio: props.studio,
    showStyleBase: props.showStyleBase
  }

  interpolatePanelStrings(..., interpolateSource)

  const configSource: ConfigPropsSource = {
    studio: props.studio,
    showStyleBase: props.showStyleBase
  }

  assertConfigSource(configSource)
}
```
but most likely something like:
```
someFunction(props) => {
  const studioAndShowStyle: StudioAndShowStyle = {
    studio: props.studio,
    showStyleBase: props.showStyleBase
  }

  interpolatePanelStrings(..., studioAndShowStyle)
  assertConfigSource(studioAndShowStyle)
}
```
// Disregard the horrible names I have used for my examples